documentation/tutorials/text-search-semantic.md

```diff
@@ -0,0 +1,317 @@
+---
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Semantic Text Search: Quick start "
+---
+
+The main goal of this tutorial is to investigate and report on the possibility of providing out of the box semantic search capabilities to text applications with as little application specific tunning as possible. In order to do that, we will first describe the data used, the metrics of interest and important baselines that will be used to judge our results. 
+
+Once that is done we will show how to use pre-trained text embeddings to match documents based on the distance between query and document embeddings by using the Vespa operator that implements nearest neighbor search. It is shown that the pre-trained embeddings indeed carry significant information about query-document relevance. 
+
+However, in an unexpected turn of events, our analysis start to show that the [MS MARCO dataset](https://microsoft.github.io/msmarco/) is more biased toward term-matching signals than we thought was the case at the beginning of our experiments. We try our best to show the evidence we found to support this claim and are open to discuss with the research community and re-evaluate this conclusion based on feedback.
```

Comment:
Again, is this a tutorial or a report? Perhaps state here that an additional goal of the tutorial is to recreate the results showing bias of term-matching signals in this dataset? That is valuable to readers as its some form of analysis they can bring to their own work.